src/config.rs

```diff
@@ -497,6 +497,8 @@ pub(crate) struct BotPullRequests {}
 #[serde(deny_unknown_fields)]
 pub(crate) struct RenderedLinkConfig {
```

Comment:
this struct is public, I would add a little documentation about its field (although the name of the fields are sort-of self-explanatory)

Reply:
Proper documentation for the handler and it's fields are at https://forge.rust-lang.org/triagebot/rendered-link.html

Reply:
I like having it while exploring the code but ... ok, I guess it's a matter of taste